nucliadb/nucliadb/search/search/query.py

```diff
@@ -48,6 +48,7 @@ async def global_query_to_pb(
     fields: Optional[List[str]] = None,
     sort: Optional[str] = None,
     sort_ord: int = Sort.ASC.value,
+    sort_limit: int = 100,
```

Comment:
Should we cap it to some value no matter what the user input?

Reply:
It might be a good idea, but I'm not sure what should be that limit...

Reply:
Some thoughts:

I wonder if the sort_limit should not be exposed as an endpoint parameter... maybe it makes sense for the sort_limit to be somehow compared to the page_size that the user is requesting.

For instance: if the user is requesting 100 results + sorting, we calculate sort.limit as 100 + (100 * 0.5) 

This way, if we enforce a limit to page_size (which we probably should), then we also have an upper bound for sort.limit